DATA/production/qc-async/mid.json

```diff
@@ -31,10 +31,12 @@
         "detectorName": "MID",
         "cycleDurationSeconds": "60",
         "maxNumberCycles": "-1",
+        "mergerCycleMultiplier": "5",
+        "resetAfterCycles": "1",
```

Comment:
Indeed this does not make sense in async, it will have no effect. What would you like to achieve by this setting?

Reply:
Well, the main purpose of the change was to re-activate one check with the same parameters that we use in sync. I have to say I'm not expert of QC, so I simply copy-pasted the `mergerCycleMultiplier` and `resetAfterCycles`. But If they make no sense in async I will remove them